docs/faq.md

```diff
@@ -38,9 +38,9 @@ For example, let's say that a preset instead used the webpack object configurati
 config.plugins.push(new webpack.EnvironmentPlugin(['NODE_ENV']));
 ```
 
-If you wanted to extend this plugin in your own project to add more environment variables, you would most likely
+If you wanted to extend this plugin in your project to add more environment variables, you would most likely
 resort to either adding a new instance of the `EnvironmentPlugin` by requiring webpack yourself, or looping through
-webpack's plugin array, removing the plugin, and re-instantiating it with your own arguments.
+webpack's plugin array, removing the plugin, and re-instantiating it with your arguments.
```

Comment:
I think the existing wording was better here, as mentioned before:
https://github.com/neutrinojs/neutrino/pull/1592#discussion_r456028828

"own" is referring to two different things, so isn't duplicating, and it's important to the meaning of the sentence. The key part of this paragraph is that how you use code in your "own" project, has different needs from a "shared" project.

Please don't close review comments as "resolved" just because you disagree. That doesn't mean you can't start a discussion to convey your points and try and persuade the reviewer - but shutting down the conversation is a quick way to alienate the project maintainers :-)

Reply:
I will review the changes once again and will make the necessary changes. And about the part of me marking the comments as 'resolved' , I am sorry for that as I am quite new and is trying to learn the environment of GitHub, thank you for correcting, I will keep that in mind for future instances.